services/src/datasets/external/aruna/mod.rs

```diff
@@ -486,6 +488,7 @@ impl ArunaDataProvider {
                 allow_alphaband_as_mask: true,
                 retry: None,
             }),
+            cache_ttl: CacheTtl::Seconds(15 * 60), // the download-link validity
```

Comment:
could we do a constant here?

Reply:
I changed this to using the specified ttl in the provider definition: 5c04d280f009ec129d8b8fb5a92e3fc1ca61c65b